examples/stock/company.vy

```diff
@@ -154,3 +136,21 @@ def debt() -> uint256:
 @external
 def worth() -> uint256:
     return self.balance - self._debt()
+
+# Return the amount in wei that a company has raised in stock offerings.
+@view
+@internal
+def _debt() -> uint256:
+    return (self.totalShares - self._stockAvailable()) * self.price
+
+# Find out how much stock the company holds
+@view
+@internal
+def _stockAvailable() -> uint256:
+    return self.holdings[self.company]
+
+# Find out how much stock any address (that's owned by someone) has.
+@view
+@internal
+def _getHolding(_stockholder: address) -> uint256:
+    return self.holdings[_stockholder]
```

Comment:
These two methods aren't really that useful

Reply:
true, i think they are good there just to demonstrate that internal functions can be defined after they are called